backend/src/main/java/de/neuefische/backend/model/openai/ChatGptRequest.java

```diff
@@ -0,0 +1,32 @@
+package de.neuefische.backend.model.openai;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class ChatGptRequest {
+    /**
+     {
+     "model": "gpt-3.5-turbo",
+     "response_format": { "type": "json_object" },
+     "messages": [{"role": "user", "content": "Say this is a test!"}],
+
+     }
+     */
```

Comment:
you dont want comments in your production code

Reply:
done!